docs/topics/data_calculation.md

```diff
@@ -100,15 +106,15 @@ or the same entity in NGSIv2 format:
          }
       }
 
-To see the complete set of operations and features available for the Expression Language, refer to
-the [specification](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/expressionLanguage.md).
+To see the complete set of operations and features available for JEXL, refer to
+the [specification](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#expression-language-support).
 
 ## Calculations based on data in Context Entities
 
 For those data values that doesn't come from devices, but from external systems, and for those cases when the sources of
 context information to calculate those values are multiple, the [CEP](../cep.md) can be used to make some of those calculations.
 
-The CEP gets a notification cointaining a configurable set of attributes (as set in the subscription) and that allows using more fields than the ones sent by device. The notification mechanism allows to send the previous value of an attribute as metadata `previousValue` (see [metadata in notifications](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#builtin-metadata)). So, as an example, we can think of an entity that periodically sends its coordinates and the time of the measure. We could calculate its average velocity by means of a rule that updates its field `velocity`.
+The CEP gets a notification cointaining a configurable set of attributes (as set in the subscription) and that allows using more fields than the ones sent by device. The CEP allows applying EPL (Esper Processing Language) rules, which can also include calculations on the received data. The notification mechanism allows to send the previous value of an attribute as metadata `previousValue` (see [metadata in notifications](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#builtin-metadata)). So, as an example, we can think of an entity that periodically sends its coordinates and the time of the measure. We could calculate its average velocity by means of a rule that updates its field `velocity`.
```

Comment:
Maybe just add a link to EPL doc of perseo: https://github.com/telefonicaid/perseo-fe/blob/master/docs/API/plain_rules.md#epl-text